### PR TITLE
Updating a request/response Context should lock mutex for read/writing.

### PR DESCRIPTION
**Pull request by pytlesk4**

### CODE REVIEW

**Comment by pytlesk4** [comment]
I signed it!

File: martianhttp/authority_handler.go [hunk 1]
```diff
@@ -39,4 +39,5 @@ func NewAuthorityHandler(ca *x509.Certificate) http.Handler {
 func (h *authorityHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	rw.Header().Set("Content-Type", "application/x-x509-ca-cert")
 	rw.Write(h.cert)
+	rw.WriteHeader(200)
```
Comment (pytlesk4) [inline martianhttp/authority_handler.go]: Not a problem, I will undo that.

**Comment by pytlesk4** [comment]
Done!

**Comment by bramhaghosh** [comment]
Thanks for the PR!

**Comment by bramhaghosh** [comment]
@pytlesk4 can you confirm that you have signed the CLA at
https://cla.developers.google.com/

I've reverted this until we can confirm - thanks!

**Comment by pytlesk4** [comment]
image

Confirmed, I used my work email for the CLA. I can use my other email, that is associated with this account.

**Comment by bramhaghosh** [comment]
Oh that's what the issue is. You need to use the same email that's
associated with the git commits.

**Comment by bramhaghosh** [comment]
Can you sign the CLA with your other email as well? Maybe your git client
was configured with your personal email address.

**Comment by pytlesk4** [comment]
Just submitted a new one.

**Comment by bramhaghosh** [comment]
we can reopen and merge this

**Pull request merged**